pages/common/fswatch.md

```diff
@@ -0,0 +1,15 @@
+# fswatch
+
+> A cross-platform file change monitor. 
+
+- Run a bash command on file creation, update or deletion:
+
+`fswatch -0 {{path_to_watch}} | xargs -0 {{bash_command}}`
+
+- Use `{}` in your bash command as a placeholder for the absolute path to the file change:
+
+`fswatch -0 {{path_to_watch}} | xargs -0 -I {} {{bash_command}}`
+
+- Filter by certain event type, eg. Updated, Deleted or Created:
+
+fswatch -0 --event {{event_type}} {{path_to_watch}} | xargs -0 {{bash_command}}`
```

Comment:
Please mention an actual event type(Updated/Deleted/Created) instead of `event_type`.   Enhances the example.